packages/analytics/src/updateAppConfig.ts

```diff
@@ -0,0 +1,516 @@
+import dotenv from 'dotenv'
```

Comment:
we we should centralise this file and the one in gameserver into appconfig.ts itself such that it is self contained